src/module-elasticsuite-catalog/Helper/AbstractAttribute.php

```diff
@@ -272,7 +272,7 @@ public function getFilterField(AttributeInterface $attribute)
     private function prepareSimpleIndexAttributeValue(AttributeInterface $attribute, $value)
     {
         if ($this->getFieldType($attribute) == FieldInterface::FIELD_TYPE_BOOLEAN) {
-            $value = boolval($value);
+            $value = intval($value);
```

Comment:
Not sure Elasticsearch will not complain if we try indexing 0 or 1 into a boolean field.